includes/admin/class-sensei-onboarding.php

```diff
@@ -155,4 +163,85 @@ public function add_onboarding_help_tab( $screen ) {
 			]
 		);
 	}
+	/**
+	 * Register REST API route.
+	 */
+	public function register_rest_api() {
```

Comment:
We should look at the REST API architecture. I think it would be best to use the pattern of of REST API Controller classes to create the endpoints. Open to other options, of course. See WCPC for an example of how we architected it there.